### DISCUSSION TITLE
c3lang in exercism

### DISCUSSION THREAD

**Discussion by ghost** [c3lang/c3c#596] [Q&A] [answered]
Hi.

I'm thinking of creating exercises from basic to intermediate in c3lang in exercism. What do you all think about it?

<br/>

**What is [exercism](https://exercism.org/tracks)?**
*" We're building the most effective, intrinsically rewarding, and friendly place to both learn and master programming. And its free."*

<br/>

I would be happy if someone answers

**Comment by lerno** [comment, marked as answer]
That would be very nice if you want to take the time.
